articles/integrate-tailwindcss-into-flask/index.md

```diff
@@ -0,0 +1,232 @@
+<img src="img/Asset 160.png" class="h-16 w-12 mr-20 ml-8 mt-10" alt="Image">
+<div class="w-3/5 mr-0">
+<h2 class="text-gray-800 font-bold">Buzz</h2>
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore et excepturi autem iure molestias doloribus ipsa praesentium.</p>
+</div>
+</div>
+<div class="transition duration-500 ease-in-out flex py-3 px-2 hover:bg-blue-400 transform hover:-translate-y-1 hover:scale-110 hover:text-white hover:shadow-none hover:rounded border border-gray-300 shadow-lg"class="transition duration-500 ease-in-out flex py-3 px-2 hover:bg-blue-400 transform hover:-translate-y-1 hover:scale-110 hover:text-white hover:shadow-none hover:rounded border border-gray-300 shadow-lg">
+<img src="img/logo.jpg" class="h-16 w-12 mr-20 ml-8 mt-10" alt="Image">
+<div class="w-3/5 mr-0">
+<h2 class="text-gray-800 font-bold">Buzz</h2>
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore et excepturi autem iure molestias doloribus ipsa praesentium.</p>
+</div>
+</div>
+<div class="transition duration-500 ease-in-out flex py-3 px-2 hover:bg-blue-400 transform hover:-translate-y-1 hover:scale-110 hover:text-white hover:shadow-none hover:rounded border border-gray-300 shadow-lg"class="transition duration-500 ease-in-out flex py-3 px-2 hover:bg-blue-400 transform hover:-translate-y-1 hover:scale-110 hover:text-white hover:shadow-none hover:rounded border border-gray-300 shadow-lg">
+<img src="img/Asset 150.png" class="h-16 w-12 mr-20 ml-8 mt-10" alt="Image">
+<div class="w-3/5 mr-0">
+<h2 class="text-gray-800 font-bold">Buzz</h2>
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore et excepturi autem iure molestias doloribus ipsa praesentium</p>
+</div>
+</div>
+</div>
+</div>
+
+</body>
+</html>
+```
+
+Observe the link to the CSS file points to the `css/main.css` file and not the pre-processed directives in the `src/style.css` file.
+
+```html
+<link rel="stylesheet" href="{{url_for('static',filename='css/main.css')}}">
+```
+
+## Test Running 
+Save the index file, and if the server is still running go to `localhost:5000/index`  and you should see the index page looking better with the styling as shown below:
+
+![demo2](/engineering-education/integrate-tailwindcss-into-flask/demo2.jpeg)
+
+And there you have it!. 
+
+## Conclusion
+
+In this article, we've covered how to structure a simple flask app to accomodate the TailwindCss package, we also saw how to install tailwindcss and how to generate CSS from its pre-processor directives into a file of our choosing and lastly we saw how to use it in our flask templates. At the end of this session you should now be able to install tailwindcss into your flask projects and integrate it successfully into your applications.
```

Comment:
Please add more details in the conclusion. While this conclusion is okay, the conclusion is usually a good place to give the reader more ideas on what they can do, how they can improve on the code, maybe even suggest more resources. Right now it is just redundant.

Also, replace the word session with article

Reply:
Okay I'll do that thanks